pkg/porter/credentials_test.go

```diff
@@ -295,6 +296,10 @@ func TestShowCredential_PreserveCase(t *testing.T) {
 }
 
 func TestCredentialsEdit(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip() // bash and vi not available on windows
```

Comment:
could this be achieved with powershell commands?

Reply:
yep you are right, thanks